public_suffix_list.dat

```diff
@@ -2,6 +2,12 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+// Please pull this list from, and only from https://publicsuffix.org/list/public_suffix_list.dat , 
+// rather than any other VCS sites. These sites are not intended to be used that way and will likely
+// be gone in the future.
+
```

Comment:
I'd remove the second sentence and replace it with "Pulling from any other URL is not guaranteed to be supported."